atari_py/ale_interface/src/games/supported/BankHeist.cpp

```diff
@@ -1,5 +1,5 @@
 /* *****************************************************************************
- * The lines 97, 107 and 115 are based on Xitari's code, from Google Inc.
+ * The method lives() is based on Xitari's code, from Google Inc.
```

Comment:
`lives()` is defined in `.hpp` file, do these comments need to be here (and in other `.cpp` files) ?